packages/http-specs/specs/authentication/api-key/main.tsp

```diff
@@ -1,5 +1,5 @@
 import "@typespec/http";
-import "@typespec/spec-lib";
+import "@typespec/spector";
 
 using Http;
 using SpecLib;
```

Comment:
@timotheeguerin Do we want to keep this as using SpecLib? Or using Spector?

Reply:
ah yeah good point will change too